proteus/mprans/RANS3PF.py

```diff
@@ -521,7 +525,7 @@ def attachModels(self, modelList):
                             self.ebq_global_grad_phi_s[ebN,kb,:]=sdNormals
                             for j in range(len(sdNormals)):
                                 corresponding_point_on_boundary[j] = self.model.ebq_global['x'][ebN,kb][j] - sdf_ebN_kb*sdNormals[j]
-                            self.ebq_particle_velocity_s[ebN,kb,:]=vel(0.0,corresponding_point_on_boundary)
+                            self.ebq_particle_velocity_s[ebN,kb,:] = vel(t=0.0, corresponding_point_on_boundary, self.particle_centroids[i])#YY:But self.particle_centroids[i] has not been updated?
```

Comment:
@cekees Considering the first step, because `self.particle_centroids` is initialized as `0` and the information of solid has not been passed into this module, the information here is not correct. Maybe we can move this part to `preStep` and delete similar update steps in `postStep`.  What do you think?

Reply:
If we're going to be providing analytical velocities then the centroids should also be provided a this point, shouldn't they?